service/create/keypair.go

```diff
@@ -0,0 +1,44 @@
+package create
+
+import (
+	"fmt"
+	"io/ioutil"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/ec2"
+)
+
+type keyPairInput struct {
+	ec2Client   *ec2.EC2
+	clusterName string
+}
+
+func (s *Service) publicKeyMaterial() ([]byte, error) {
+	return ioutil.ReadFile(s.pubKeyFile)
+}
+
+func (s *Service) keyPair(input keyPairInput) (string, error) {
+	pkm, err := s.publicKeyMaterial()
+	if err != nil {
+		return "", err
```

Comment:
In general we want to handle errors with `microerror.MaskAny(err)`. Debugging errors is way more easy when having the error's trace in case shit hits the fan.